### ISSUE TITLE
TypeError: Cannot read properties of undefined (reading 'position') and (reading 'id')

### ISSUE THREAD

**Issue by clabnet** [bcakmakoglu/vue-flow#265]
Adding a node to board, then try to delete it with BACKSPACE key.
Using vue-flow 0.4.35 and vite 3.0.4

## Expected Behavior
I would delete the node

## Current Behavior
Console errors
```
Cannot read properties of undefined (reading 'position')
    at _a2 (vue-flow.es.js:4218:30)
...
Cannot read properties of undefined (reading 'id')
    at vue-flow.es.js:3747:44
```

## Steps to Reproduce
[Minimal Github repo](https://github.com/clabnet/vue3-ts-vite-vueflow-naiveui)
1. Add a node to board
2. Select node
3. Press BACKSPACE key to delete it

**Comment by bcakmakoglu** [comment]
Thanks for reporting, I'll take a look :)

**Comment by bcakmakoglu** [comment]
@clabnet Fix released with `v0.4.36`
